<?xml version="1.0" encoding="UTF-8"?><testrun name="UserTaskTest" project="TaskManagmentSystem" tests="3" started="3" failures="0" errors="1" ignored="0">
  <testsuite name="com.example.www.customTest.UserTaskTest" time="2.614" displayname="UserTaskTest" uniqueid="[engine:junit-jupiter]/[class:com.example.www.customTest.UserTaskTest]">
    <testcase name="testUpdateTask" classname="com.example.www.customTest.UserTaskTest" time="2.559" displayname="testUpdateTask()" uniqueid="[engine:junit-jupiter]/[class:com.example.www.customTest.UserTaskTest]/[method:testUpdateTask()]">
      <error>org.mockito.exceptions.misusing.PotentialStubbingProblem: 
Strict stubbing argument mismatch. Please check:
 - this invocation of 'toDto' method:
    taskMapper.toDto(
    com.example.www.entity.Task@de18f63
);
    -&gt; at com.example.www.service.TaskService.updateTask(TaskService.java:82)
 - has following stubbing(s) with different arguments:
    1. taskMapper.toDto(
    com.example.www.entity.Task@29a4f594
);
      -&gt; at com.example.www.customTest.UserTaskTest.testUpdateTask(UserTaskTest.java:86)
Typically, stubbing argument mismatch indicates user mistake when writing tests.
Mockito fails early so that you can debug potential problem easily.
However, there are legit scenarios when this exception generates false negative signal:
  - stubbing the same method multiple times using 'given().will()' or 'when().then()' API
    Please use 'will().given()' or 'doReturn().when()' API for stubbing.
  - stubbed method is intentionally invoked with different arguments by code under test
    Please use default or 'silent' JUnit Rule (equivalent of Strictness.LENIENT).
For more information see javadoc for PotentialStubbingProblem class.&#13;
	at com.example.www.mapper.TaskMapper.toDto(TaskMapper.java:28)&#13;
	at com.example.www.service.TaskService.updateTask(TaskService.java:82)&#13;
	at com.example.www.customTest.UserTaskTest.testUpdateTask(UserTaskTest.java:89)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testGetAllTask" classname="com.example.www.customTest.UserTaskTest" time="0.03" displayname="testGetAllTask()" uniqueid="[engine:junit-jupiter]/[class:com.example.www.customTest.UserTaskTest]/[method:testGetAllTask()]"/>
    <testcase name="testCreateTask" classname="com.example.www.customTest.UserTaskTest" time="0.014" displayname="testCreateTask()" uniqueid="[engine:junit-jupiter]/[class:com.example.www.customTest.UserTaskTest]/[method:testCreateTask()]"/>
  </testsuite>
</testrun>
